tests/Mailer/Ownership/SendMailsForOwnershipEventHandlerTest.php

```diff
@@ -35,21 +41,11 @@ protected function setUp(): void
     }
 
     /** @test
+     * @dataProvider eventToCommandProvider
      * This is the happy path
      * */
-    public function it_handles_ownership_requested_event(): void
+    public function it_handles_ownership_requested_event(Serializable $event, AbstractSendOwnershipMail $command): void
```

Comment:
I don't think this method name is correct

Reply:
It is the data provider for a test that checks if an Event (OwnershipApproved) is converted to a command "SendOwnershipApprovedEmail".
I renamed it to a generic "dataprovider" if it is confusing.